### PR TITLE
[Case Summary PDF] corrects

### PR DESCRIPTION
**Pull request by rusllonrails**
[Trello Story](https://trello.com/c/L3WHDJ3n/252-case-summary-pdf-changes)

* Applicant: Should interpolate the application name
* The RAG: The text should say what it displays on the screen (recommended etc)
* Continuous - typo
* Capitalise International Trade (the T)

### CODE REVIEW

**Comment by rusllonrails** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;L3WHDJ3n&#x2F;252-case-summary-pdf-changes"> Case summary pdf changes</a>

File: app/pdf_generators/shared_pdf_helpers/draw_elements.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def render_award_general_information(x_coord, y_coord)
   end
 
   def render_award_title(x_coord, y_coord)
-    pdf_doc.text_box form_answer.award_type_full_name.capitalize,
+    pdf_doc.text_box form_answer.award_type_full_name.camelcase,
```
Comment (matthewford) [inline app/pdf_generators/shared_pdf_helpers/draw_elements.rb]: Think `titlecase` will also work

Comment (rusllonrails) [inline app/pdf_generators/shared_pdf_helpers/draw_elements.rb]: Thanks, yeah - agree. Gonna fix

**Comment by rusllonrails** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;EN78BdMz&#x2F;442-move-apps-to-circleci"> Move Apps to circleCI</a>

**Comment by matthewford** [comment]
Can we squash the commits?

**Comment by rusllonrails** [comment]
Done

**Pull request merged**